packages/astro/src/@types/astro.ts

```diff
@@ -2191,135 +2191,70 @@ export interface AstroUserConfig {
+		 * The Content Layer API is a new way to handle content and data in Astro. It is similar to and builds upon [content collections](/en/guides/content-collections/), taking them beyond local files in `src/content` and allowing you to fetch content from anywhere, including remote APIs, or files anywhere in your project by adding a `loader` to your collection.
 		 *
-		 * #### Enabling the Content Layer API
+		 * Your existing content collections can be [migrated to the Content Layer API](#migrating-a-content-collection-to-content-layer) with a few small changes. However, it is not necessary to update all your collections at once to add a new collection powered by the Content Layer API. You may have collections using both the existing and new APIs defined in `src/content/config.ts` at the same time.
+		 *
+		 * The Content Layer API is designed to be more powerful and more performant, helping sites scale to thousands of pages. Data is cached between builds and updated incrementally. Markdown parsing is also 5-10 times faster, with similar scale reductions in memory.
 		 *
 		 * To enable, add the `contentLayer` flag to the `experimental` object in your Astro config:
 		 *
 		 * ```js
+		 * // astro.config.mjs
 		 * {
 		 * 	experimental: {
 		 * 		contentLayer: true,
 		 * 	}
 		 * }
 		 * ```
 		 *
-		 * #### Using the Content Layer API
+		 * #### Fetching data with a `loader`
 		 *
-		 * :::tip
-		 * The Content Layer API is a new way to define content collections, but many of the APIs are the same. It will be helpful to refer to the current [content collection docs](https://docs.astro.build/en/guides/content-collections/) for more information. Any differences in the API usage are highlighted below.
-		 * :::
+		 * The Content Layer API allows you to fetch your content from outside of the `src/content/` folder (whether stored locally in your project or remotely), and uses a `loader` property to retrieve your data.
 		 *
-		 * To use the Content Layer API, create a collection in `src/content/config.ts` with a `loader` property. For local files where there is one entry per file, use the `glob()` loader. You can put your content files anywhere, but *not* in `src/content` because these would be handled by the current content collections APIs instead. In this example the files are in `src/data`.
+		 * The `loader` is defined in the collection's schema and returns an array of entries. Astro provides two built-in loader functions (`glob()` and `file()`) for fetching your local content, as well as access to the API to [construct your own loader and fetch remote data](#creating-a-loader).
 		 *
-		 * ```ts
+		 * The `glob()` loader creates entries from directories of Markdown, MDX or JSON files from anywhere on the filesystem. It accepts a `pattern` of entry files to match, and a `base` file path of where your files are located. Use this when you have one file per entry.
```

Comment:
```suggestion
		 * The `glob()` loader creates entries from directories of Markdown, MDX, Markdoc, or JSON files from anywhere on the filesystem. It accepts a `pattern` of entry files to match, and a `base` file path of where your files are located. Use this when you have one file per entry.
```
If we're going to be there in time!

Also just checking, "anywhere on the filesystem" feels like it might be interpreted to include local files on your machine, not just in your project. If this is true, great! Otherwise, we could consider "from anywhere in your project." if that's the limitation.

Reply:
Yup. Anywhere at all!